client/me/purchases/cancel-purchase/style.scss

```diff
@@ -1,96 +1,195 @@
+.is-section-me.is-global-sidebar-visible .purchases__cancel-domain.main {
+	height: calc(100vh - var(--masterbar-height) - var(--content-padding-top) - var(--content-padding-bottom));
+}
+
+.cancel-purchase__wrapper-card {
+	padding: 32px;
+	padding-top: 12px;
+
+	.purchases__cancel & {
+		box-shadow: none;
+		padding: 0;
 	}
+}
 
-	p {
-		color: var(--color-neutral-40);
-		font-size: $font-body-small;
+.cancel-purchase__back .header-cake__back,
+.confirm-cancel-domain__back .header-cake__back {
+	padding: 24px 0;
+	text-align: left;
+}
+
+.cancel-purchase__right .purchases-site__header.card {
+	background: var(--studio-gray-0);
+	border-radius: 2px;
+
+	.site .site__content {
+		padding: 24px 18px;
+
+		@include break-mobile {
+			padding: 12px;
+		}
 	}
+}
 
-	hr {
-		background: var(--color-neutral-0);
+.cancel-purchase__support-link {
+	color: var(--color-text-subtle);
+	font-size: $font-body-small;
+	margin-top: 12px;
+
+	.is-link {
+		font-size: $font-body-small;
+		text-decoration: none;
 	}
 }
 
-.cancel-purchase__content {
-	margin: 0;
-	padding: 0 0 10px;
+.cancel-purchase__domain-options {
+	.card {
+		margin: 16px 8px;
+	}
 
-	@include breakpoint-deprecated( ">660px" ) {
-		width: 55%;
+	.cancel-purchase__refund-domain-info {
+		color: var(--color-text-subtle);
+		display: inline-block;
+		font-size: $font-body-extra-small;
+		line-height: 1.4;
+		margin-top: 4px;
 	}
 
-	p {
-		color: var(--color-neutral-light);
-		display: block;
+	.cancel-purchase__domain-warning {
+		display: inline-block;
 		font-size: $font-body-small;
-		/* stylelint-disable-next-line declaration-property-unit-allowed-list */
-		line-height: 150%;
-		margin-top: 5px;
+		line-height: 1.4;
+
+		.form-label {
+			margin: 1.5em 0;
+		}
 	}
+}
 
-	hr {
-		margin: 18px 0;
+.cancel-purchase__inner-wrapper {
+	display: flex;
+	flex-direction: column-reverse;
+	margin-bottom: 24px;
+
+	@include break-large {
+		display: grid;
+		grid-template-columns: 7fr 3fr;
+		gap: 32px;
 	}
 }
 
-.cancel-purchase__product-information .product-link {
-	font-size: $font-body-small;
+.confirm-cancel-domain__formatted-header.formatted-header,
+.cancel-purchase__formatted-header.formatted-header {
+	margin: 16px 0;
 }
 
-.cancel-purchase__purchase-name {
-	font-size: $font-body;
-	font-weight: 600;
+.cancel-purchase__features {
+	ul.cancel-purchase__features-list {
+		list-style: none;
+		margin: 1.5em;
+		margin-bottom: 0;
+
+		@include break-mobile {
+			column-count: 2;
+			column-gap: 20px;
+		}
+
+		li {
+			list-style: none;
+			display: flex;
+			margin-bottom: 12px;
+			font-size: $font-body-small;
+
+			span {
+				margin: auto 0;
+				flex: 1;
+			}
+
+			.gridicon {
+				color: var(--color-error-40);
+			}
+		}
+	}
 }
 
-.cancel-purchase__refund-information,
-.cancel-purchase__plan-description,
-.cancel-purchase__support-information {
-	font-size: $font-body-small;
+.cancel-purchase__features-link {
+	text-align: right;
```

Comment:
Might make more sense to left or center align this. When it is aligned right it feels like it is floating out of alignment compared to the featured list items.

If you use flexbox here you can more easily account for RTL layouts too:

```
.cancel-purchase__features-link {
    display: flex;
    justify-content: flex-start; // or center
    margin-top: 1em;
}
```

Reply:
Makes sense! I'm curious as to whether you'd recommend centre-aligning or left-aligning this? 

**Centre:**
<img width="1059" alt="Screenshot 2024-09-11 at 21 54 37" src="https://github.com/user-attachments/assets/15d478c6-5603-4a71-b831-7c4c4039f662">

**Left:**
<img width="1062" alt="Screenshot 2024-09-11 at 21 55 28" src="https://github.com/user-attachments/assets/0bd4c51d-5358-47be-b4b5-9b6c33cdd5e9">

Just an idea, but one thing I feel with both the above options is that the text feels a bit crowded with the link. To avoid this, I quite like the idea of treating the features list almost like its own card, but that might just be my preference.  

<img width="1063" alt="Screenshot 2024-09-11 at 21 54 04" src="https://github.com/user-attachments/assets/891624e2-afcf-4b2f-9a1c-b07f863017f1">

Let me know what you think (and thank you for going through and summarising all the feedback!). :)